src/EB/SDK/SuppressionImport/SuppressedRecipientFactory.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * RecipientFactory.
+ *
+ * Unauthorized copying or dissemination of this file, via any medium is strictly prohibited
+ * Proprietary and confidential
+ *
+ * @author     Fernando Martins <fernando.martins@wondeotec.com>
+ * @copyright  Copyright (C) Wondeotec SA - All Rights Reserved
+ * @license    LICENSE.txt
+ */
+namespace EB\SDK\SuppressionImport;
+
+/**
+ * EB\SDK\SuppressionImport\RecipientFactory
+ */
+class SuppressedRecipientFactory
+{
+    /**
+     * Creates a recipient with the mandatory information.
+     *
+     * @param string $value The recipient email address
+     *
+     * @return SuppressedRecipient
+     */
+    public static function createSimpleSuppressedRecipient($value)
```

Comment:
if the value is the recipient email address, then we can call it emailAddress instead $value, no?